### PR TITLE
(IA-3829) Code style glowup: break long line

### PR DESCRIPTION
**Pull request by mlilynolting**
Reopening this issue for a code prettification at Miguel's request! Breaking a long line on the ternary and an internal array.

Reran this locally to sanity check.

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]

**Review by msilva-broad — APPROVED** [review]
looks great!  thx for squeezing in the cleanup :)

File: src/pages/workspaces/workspace/analysis/modals/ComputeModal.js [hunk 1]
```diff
@@ -1490,8 +1490,15 @@ export const ComputeModalBase = ({
...
           ])],
           [willRequireDowntime(), () => h(Fragment, [
-            existingRuntime && existingRuntime.tool !== desiredTool ? p(['By continuing, you will be changing the application of your cloud environment from ', strong([existingRuntime.tool]), ' to ',
-        desiredToolPhrase, '.']) : undefined,
+            existingRuntime && existingRuntime.tool !== desiredTool ?
+              p([
+                'By continuing, you will be changing the application of your cloud environment from ',
+                strong([existingRuntime.tool]),
+                ' to ',
+                desiredToolPhrase,
```
Comment (jdcanas) [inline src/pages/workspaces/workspace/analysis/modals/ComputeModal.js:1498]: might be a bit more concise to condense the last three strings into one when its split up, like this 

` to ${desiredToolPhrase}.`

**Review by jdcanas — APPROVED** [review]

File: src/pages/workspaces/workspace/analysis/modals/ComputeModal.js (hunk 1 above)
Comment (mlilynolting) [reply src/pages/workspaces/workspace/analysis/modals/ComputeModal.js:1498]: Noted for next time! Would this be something I'd do with string interpolation (backticks) or does the react-hyperscript-helpers library handle interpolation its own way?

Comment (kyuksel) [reply src/pages/workspaces/workspace/analysis/modals/ComputeModal.js:1498]: It'd be via string interpolation (i.e. `to ${desiredToolPhrase}.` ).

**Pull request merged**